sdk/communication/Azure.Communication.CallAutomation/src/Generated/CallConnectionRestClient.cs

```diff
@@ -598,7 +598,7 @@ public async Task<Response<MuteParticipantResult>> MuteAsync(string callConnecti
             await _pipeline.SendAsync(message, cancellationToken).ConfigureAwait(false);
             switch (message.Response.Status)
             {
-                case 200:
+                case 202:
```

Comment:
It is not related to Dialog, however it is a change that is from the swagger (and the service returns 202 instead of 200) and the tests and build will fail without this update.